pom.xml

```diff
@@ -306,20 +313,17 @@
                 <executions>
                     <execution>
                         <id>compile</id>
-                        <phase>compile</phase>
                         <goals>
                             <goal>compile</goal>
                         </goals>
                         <configuration>
                             <sourceDirs>
                                 <source>src/main/java</source>
-                                <source>target/generated-sources/swagger/oppgave</source>
```

Comment:
Hvorfor fjernes generert kode for oppgave?

Reply:
Fikset